libraries/standard/http/src/http_client.c

```diff
@@ -108,19 +107,80 @@ static HTTPStatus_t _getFinalResponseStatus( HTTPParsingState_t parsingState,
 static HTTPStatus_t _receiveAndParseHttpResponse( const HTTPTransportInterface_t * pTransport,
                                                   HTTPResponse_t * pResponse );
 
+/**
+ * @brief Converts an integer value to its ASCII representation in the passed buffer.
+ *
+ * @param[in] value The value to convert to ASCII.
+ * @param[out] pBuffer The buffer to store the ASCII representation of the integer.
+ *
+ * @return Returns the number of bytes written to @p pBuffer.
+ */
+static uint8_t _convertInt32ToAscii( int32_t value,
+                                     char * pBuffer,
+                                     size_t bufferlength );
+
+/*-----------------------------------------------------------*/
+
+static uint8_t _convertInt32ToAscii( int32_t value,
```

Comment:
Couldn't we also use itoa and define a separate method for returning the length of an int32_t?

Reply:
`itoa` is not part of ANSI C as mentioned under the Portability section of its reference doc: http://www.cplusplus.com/reference/cstdlib/itoa/

We only want to add dependencies on ANSI C defined functions